src/main/java/top/seraphjack/simplelogin/server/capability/CapabilityPassword.java

```diff
@@ -66,11 +81,21 @@ public void setGameType(int gameType) {
         public int getGameType() {
             return this.gameType;
         }
+
+        @Override
+        public Position getLastPosition() {
+            return pos != null ? pos : SLConstants.defaultPosition;
```

Comment:
If the default implementation is added, then it can be:
```java
if (pos != null) {
    return pos;
}
return ISLEntry.super.getLastPosition();
```

Reply:
No, no default in the interface.